components/support/rs_client/src/http_client.rs

```diff
@@ -59,14 +59,30 @@ impl Client {
             "v1/buckets/{}/collections/{}/records",
             &self.bucket_name, &self.collection_name
         );
-        let url = self.base_url.join(&path)?;
-        let req = Request::get(url);
-        self.make_request(req)
+        self.make_request(path)
+    }
+
+    pub fn get_attachment(&self, attachment_location: &str) -> Result<Response> {
+        let path = format!(
+            "attachments/{}/{}/{}",
+            &self.bucket_name, &self.collection_name, attachment_location
+        );
+        self.make_request(path)
     }
 
-    fn make_request(&self, request: Request) -> Result<Response> {
+    pub fn get_records_since(&self, last_modified: i64) -> Result<Response> {
```

Comment:
ah, I see you have it here but not in the UDL - my comment above is about how you might communicate the timestamp on the first call.

Reply:
Oh thanks for the reminder, this should be added now!